### ISSUE TITLE
GetNextSurveyStep is not a function TypeError after launching recurring survey. 

### ISSUE THREAD

**Issue by wouter-leistra** [PostHog/posthog-js#1258]
I created a recurring survey for every 30 days, after launch our Sentry started reporting:

```
TypeError onNextButtonClick(src/extensions/surveys)

New

Level: Error

Unhandled

c.getNextSurveyStep is not a function

```

We have version 1.139.2 running in our Web application.

**Comment by kiranP007** [comment]
We are also getting similar issue, we have 1.128.1 posthog-js version.

**Comment by neilkakkar** [comment]
Hey @wouter-leistra sorry about this :/ - we'll push a fix shortly, meanwhile you can fix this by upgrading to 1.139.3 or higher

**Comment by neilkakkar** [comment]
should be fixed with https://github.com/PostHog/posthog-js/pull/1260 - let me know if you're still facing issues, irrespective of version
